components/elm/src/dyn_subgrid/dynHarvestMod.F90

```diff
@@ -66,22 +65,23 @@ module dynHarvestMod
   ! for FATES: capacity for passing harvest data in units of carbon harvested per year (per grid cell) has been added
   ! but these data are not yet included in the input file
   ! the code here can be changed to wood_harvest_units = harvest_carbon to pass carbon data to FATES if:
-  !  the carbon data are in the same input variables as listed below
-  !  and the carbon units in the input file match that expected by FATES
+  ! the carbon data are in the same input variables as listed below
+  ! and the carbon units in the input file match that expected by FATES
 
   integer, public, parameter :: num_harvest_vars = 5
   character(len=64), public, parameter :: harvest_varnames(num_harvest_vars) = &
-       [character(len=64)  :: 'HARVEST_VH1', 'HARVEST_VH2', 'HARVEST_SH1', 'HARVEST_SH2', 'HARVEST_SH3']
+       [character(len=64) :: 'HARVEST_VH1', 'HARVEST_VH2', 'HARVEST_SH1', 'HARVEST_SH2', 'HARVEST_SH3']
+
+  type(dyn_var_time_uninterp_type) :: harvest_vars(num_harvest_vars)   ! value of each harvest variable
+
   ! the units flag must match the units of harvest_varnames
   ! set this here because dynHarvest_init is called after alm_fates%init
   ! this flag is accessed only if namelist do_harvest is TRUE
-  integer, public          :: wood_harvest_units = hlm_harvest_area_fraction
-  
-  type(dyn_var_time_uninterp_type) :: harvest_vars(num_harvest_vars)   ! value of each harvest variable
 
-  real(r8) , allocatable, public   :: harvest_rates(:,:) ! category harvest rates (d1) in each gridcell (d2)
+  integer, public, parameter    :: wood_harvest_units = 2    ! 1 = area fraction, 2 = carbon
```

Comment:
@sshu88, is the long-term plan to have wood_harvest_units read in or defined in the file? or set in a namelist?

Reply:
In my opinion it would be better to set `wood_harvest_units` as a namelist option. However, this option may require the change of another set of harvest rate data in different unit.

Reply:
I agree that a namelist option would be best in the long-term, particularly if the input file eventually contains both carbon and area prescriptions.